includes/export/abstract-wc-csv-exporter.php

```diff
@@ -236,7 +235,7 @@ protected function get_csv_data() {
 	protected function export_column_headers() {
 		$columns    = $this->get_column_names();
 		$export_row = array();
-		$buffer     = fopen( 'php://output', 'w' );
+		$buffer     = fopen( 'php://output', 'w' ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_read_fopen
```

Comment:
Should we use the recommended alternative instead of ignoring the rule?

Reply:
For php://output this might be a false identify, with php://output you are basically just writing to the output stream of the browser and not an actual file so think this is safe to use.